commands/FBPrintCommands.py

```diff
@@ -299,3 +300,15 @@ def run(self, arguments, options):
       object = fb.evaluateObjectExpression(objectToMessage)
       printCommand = 'po [{} valueForKeyPath:@"{}"]'.format(object, keypath)
       lldb.debugger.HandleCommand(printCommand)
+
+
+class FBPrintApplicationDocumentsPath(fb.FBCommand):
+  def name(self):
+    return 'ppath'
```

Comment:
I feel like `pdocspath` might be a better command here, even if it is a bit verbose. Luckily the prefix matching will then allow `pdocs` to work as well. Thoughts?